### ISSUE TITLE
Add organisation's mission in readme.md file

### ISSUE THREAD

**Issue by AkashsRepositories** [devnest-coders/dsa-practice#15]

**Comment by mehraankush** [comment]
can you tell me more about the org.

**Comment by Sarabjeet108** [comment]
I would like to work on this issue. Please assign it to me.

**Comment by ANKIT9761** [comment]
Yup so our org mission is to provide free learning resources to persons who are willing to learn and provide them the environment to learn and grow.
Our main focus is peer learning and working and studying in groups.
We focus on consistent efforts than a single night fight or 10-15 days motivation based learning.
We aim to provide positive learning environment filled with diverse people who can bring in their own special powers to help you ease and make everyone learning journey Enjoyable.
When you can learn and have fun at same time with you peers and improve your communication skills why don't you do it.
Once you are skilled enough we also provide referals through our connections or post companies information which are actively recuriting.
As our main focus is you should master a skill enough that you can make a earning out of it and use it to fuel your other passions.

Last but not least I truly believe everyone has potential to do something bigger.They sometime just need the right push or environment to get started. We are aiming for that small push and see them doing wonders and make full use of their potential.
Every little help and contribution counts so try contributing as much possible to make full use of this open source festivel which will last a month.
May the code be with us.

**Comment by ANKIT9761** [comment]
Anyone interested to work on this documentation can ping me here and i will assig that person this issue

**Comment by mehraankush** [comment]
ok i'll do my best.. assign it to me

**Comment by ANKIT9761** [comment]
ok @mehraankush thanks for the response assigned this to you

**Comment by ANKIT9761** [comment]
opening this issue again and looking for new assignee for this issue removing @mehraankush from this issue.
